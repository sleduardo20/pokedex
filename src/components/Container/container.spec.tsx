import { screen } from '@testing-library/react';
import { renderWithTheme } from 'util/renderWithTheme';

import Wrapper from '.';

describe('<Wrapper/>', () => {
  it('should be able render Wrapper correctly', () => {
    const { container } = renderWithTheme(
      <Wrapper>
        <h1>Title</h1>
        <span>Content</span>
      </Wrapper>,
    );

    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument();
    expect(screen.getByText('Content')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
