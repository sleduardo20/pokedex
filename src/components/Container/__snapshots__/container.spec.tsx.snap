// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper/> should be able render Wrapper correctly 1`] = `
.c0 {
  width: max(36rem,100%);
  height: 100vh;
  margin: 0 auto;
  background: url('/img/bgPokbola.svg') no-repeat center;
  background-color: #FFFFFF;
  background-size: cover;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  class="c0"
>
  <h1>
    Title
  </h1>
  <span>
    Content
  </span>
</div>
`;
